packages/web-components/@sfx/products/src/products.ts

```diff
@@ -0,0 +1,92 @@
+import {
+  customElement,
+  html,
+  property,
+  TemplateResult,
+} from 'lit-element';
+import Product from '@sfx/product/src/product';
+import { Base } from '@sfx/base';
+
+/**
+ * The sfx-products web component wraps and renders a number of
+ * sfx-product components. It wraps each sfx-product component in an
+ * additional wrapper for flexibility.
+ */
+@customElement('sfx-products')
+export default class Products extends Base {
+  @property({ type: Number, reflect: true }) maxItems = 12;
```

Comment:
I'm under the impression that this component would be able to act as the main product grid on a PLP. Considering that, I'm not a fan of this property living in the component. I think the number of products being displayed should be limited by where the search call is made and then the product wrapper displays whatever number of products is provided to it. 

I believe the default number of products returned from a search request is 10. However, a client will need to be able to adjust that number before the search request is made, ideally in any sort of search driver they are using (either the one in the logic layer, or one that they build) as an option. If this property also lives in the product wrapper, then it basically just needs to mirror the number before the search call and is redundant. If for some reason the pre-search call property is ever higher than the products wrapper property, then some products will fail to display (creates a possible point of confusion for an implementer).

Reply:
Thinking about it, I agree. This component should just display what it's given.

Reply:
@geoffcoutts @jmbtutor  Most of this makes sense to me, but it seems odd to not have the capability to limit the number of displayed items for a display component. It sounds like the problem here is that this forces the client to specify the information in both places. What if we default this value to `0` or `undefined`, and if it's that value, display every product it's given?

In that case, if the client specifies some number of search results to receive, they will not require specifying it here. So it wouldn't be redundant. Thoughts?

Reply:
@DanielMoniz-GBI @jmbtutor I'm still of the opinion that ultimately it should live in whatever is driving the search. Wouldn't best practice for client implementations (using this product or not) be for them to specify their product count limit before a search occurs in order to minimize data retrieval and time? 
Ex. Searching for 30 products but then only limiting it to display the first 3 retrieves a lot of unnecessary data. 

What use cases do you foresee for using a max items value in the component?